libraries/adsk/shaders/brick/ND_brick_02_shader.mtlx

```diff
@@ -1,55 +1,61 @@
+      <output name="out" type="surfaceshader" value="" />
+   </nodedef>
+   <nodegraph name="NG_brick_02_surfaceshader_template" nodedef="adsk:ND_bricks_02_surfaceshader" version="1.0" namespace="adsk">
+      <normalmap name="bricks_02_normalmap" type="vector3">
+         <input name="in" type="vector3" nodename="bricks_02_normal" />
+      </normalmap>
+      <tiledimage name="bricks_02_basecolor" type="color3">
+         <parameter name="file" type="filename" interfacename="color_image" />
+         <input name="uvtiling" type="vector2" value="2, 2" />
+      </tiledimage>
+      <tiledimage name="bricks_02_normal" type="vector3">
+         <parameter name="file" type="filename" interfacename="normalmap_image" />
+         <input name="uvtiling" type="vector2" value="2, 2" />
+      </tiledimage>
+      <tiledimage name="bricks_02_roughness" type="float">
+         <parameter name="file" type="filename" interfacename="roughness_image" />
+         <input name="uvtiling" type="vector2" value="2, 2" />
+      </tiledimage>
+      <standard_surface name="bricks_02_sts" type="surfaceshader" version="1.0.1">
+         <input name="base_color" type="color3" nodename="bricks_02_basecolor" />
+         <input name="specular_roughness" type="float" nodename="bricks_02_roughness" /> />
+         <input name="normal" type="vector3" nodename="bricks_02_normalmap" />
+      </standard_surface>
+      <output name="out" type="surfaceshader" nodename="bricks_02_sts" />
+   </nodegraph>
+
+   <nodedef name="ND_bricks_02_displacementshader" node="brick_02_displacement" nodegroup="shader" version="1.0" isdefaultversion="true" namespace="adsk">
+      <parameter name="displacement_image" type="filename" value="adsk/textures/bricks_02/8k/bricks_02_height_2k_raw.exr" />
+      <output name="out" type="displacementshader" value=""/>
+   </nodedef>
+   <nodegraph name="NG_brick_02_displacementshader_graph" nodedef="adsk:ND_bricks_02_displacementshader" version="1.0" namespace="adsk">      
+      <subtract name="bricks_02_disp_subtract" type="float">
+         <input name="in1" type="float" nodename="bricks_02_displacement" />
+         <input name="in2" type="float" value="0.5" />
+      </subtract>
+      <normalize name="bricks_02_disp_normalize" type="vector3">
+         <input name="in" type="vector3" nodename="bricks_02_disp_normal" />
+      </normalize>
+      <displacement name="bricks_02_disp_displacement" type="displacementshader">
+         <input name="displacement" type="vector3" nodename="bricks_02_disp_multiply" />
+         <input name="scale" type="float" value="5" />
+      </displacement>
+      <multiply name="bricks_02_disp_multiply" type="vector3">
+         <input name="in1" type="vector3" nodename="bricks_02_disp_normalize" />
+         <input name="in2" type="float" nodename="bricks_02_disp_subtract" />
+      </multiply>
+      <tiledimage name="bricks_02_displacement" type="float">
```

Comment:
minor: should the fallback color for this be 0.5, 0.5, 0.5? So that it does 0 displacement if no image is specified?